plugins/techdocs/src/reader/transformers/html/hooks/links.ts

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
maybe rename file `removeUnsafeLinks`, could be clearer?

Reply:
Same as [here](https://github.com/backstage/backstage/pull/11647/files#r879283785), what do you think?